backends/cpython/src/converter.rs

```diff
@@ -108,7 +108,7 @@ impl Converter {
                 TypePattern::Option(x) => x.rust_name().to_string(),
                 TypePattern::Bool => "ctypes.c_uint8".to_string(),
                 TypePattern::CChar => "ctypes.c_char".to_string(),
-                TypePattern::NamedCallback(x) => format!("callbacks.{}", safe_name(&x.fnpointer().internal_name())),
+                TypePattern::NamedCallback(x) => self.fnpointer_to_typename(x.fnpointer()),
```

Comment:
this is what fixed the dependency, I updated the other site as well so it was more consistent.